manifests/y/yt-dlp/yt-dlp/2022.05.18/yt-dlp.yt-dlp.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: yt-dlp.yt-dlp

```

Comment:
#68801 is adding another version of this package, and using the identifier YT.DLP. Looks like that's what we're going with.